backend/routes/courses.js

```diff
@@ -363,4 +363,52 @@ router.post(
   })
 );
 
+/**
+ * @api {post} /updateFutureCourse
+ * @apiDescription This endpoint will add future desired courses
+ * @apiExample {curl} Example usage:
+ * Http:
+ *	POST /courses/updateFutureCourse HTTP/1.1
+ *	Host: localhost:3001
+ *	Content-Type: application/json
+ *	   {
+ *       "student_id": "225058391",
+ *       "courses": ["ECSE 321", "ECSE 323"]
+ *     }
+ * Curl:
+ *	curl -X POST \
+ *	http://localhost:3001/courses/updateFutureCourse
+ *	-H 'Content-Type: application/json' \
+ *	-d '{
+ *       "student_id": "225058391",
+ *       "courses": ["ECSE 321", "ECSE 323"]
+ *     }'
+ *
+ * @returns true if future desired courses were added successfully
+ *          false if future desired courses were not added
+ *          invalid format future courses if the courses format is invalid
+ *          database connection handling
+ *
+ * @author: Mathieu Savoie
+ */
+router.post("/updateFutureCourse", async (req, res, next) => {
```

Comment:
I'd call this /updateDesiredCourse rather than future course because I think it's more correct. That said, it's just a suggestion so feel free to ignore that, it's a minor detail.

Reply:
done